src/RuleAsset.php

```diff
@@ -48,6 +48,10 @@ public function getTitle()
         return __('Business rules for assets');
     }
 
+    public static function getIcon()
+    {
+        return "ti ti-package";
```

Comment:
```suggestion
        return "ti ti-player-play";
```

Reply:
"ti-package" choice was to keep consistency with the menu to "Asset" menu.